.github/workflows/generate-doc-check.yml

```diff
@@ -49,4 +49,4 @@ jobs:
           sudo apt-get install graphviz
 
       - name: Compile docs for all Scala versions
-        run: sbt +compile:doc
+        run: sbt ";+compile:doc; paradox"
```

Comment:
Does this need to be `+paradox`

Reply:
I think we only need to build this once - with java 11. Paradox doesn't work with Java 17 (you need to add JPMS flags to the java command at least).